src/components/leaderboard/LeaderboardRow/style.module.scss

```diff
@@ -0,0 +1,69 @@
+@use 'src/styles/vars.scss' as vars;
+
+.row {
+  align-items: center;
+  background-color: var(--theme-surface-1);
+  display: grid;
+  font-size: 15px;
+  gap: 32px;
+  grid-template-columns: auto auto 2fr 1fr;
+  height: 64px;
+  padding: 0 32px;
+
+  &[data-style='odd'] {
+    background-color: var(--theme-background);
+  }
+}
+
+.position {
+  font-size: 20px;
+  font-weight: bold;
+  width: 40px;
+}
+
+.profilePicture {
+  border-radius: 50%;
+}
+
+.nameRank {
+  display: grid;
+  gap: 32px;
+  grid-template-columns: 1fr 1fr;
+}
+
+.name,
+.points {
+  font-weight: 500;
+}
+
+.rank {
+  color: var(--theme-text-on-surface-1);
+  opacity: 0.5;
+  text-align: center;
+}
+
+.points {
+  text-align: right;
+}
+
+@media (max-width: vars.$breakpoint-md) {
+  .row {
+    gap: 16px;
+    grid-template-columns: auto auto 1fr auto;
+    padding: 0 24px;
+  }
+
+  .nameRank {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5em;
+  }
+
+  .position {
+    margin-right: -8px;
```

Comment:
seeking to learn: what is this necessary for?

Reply:
Making the row more condensed for mobile so there's more room for the name. The position is left-aligned, so for the low numbers there's an unbalanced amount of whitespace to the right of the number, which I kind of hackily reduce for the mobile view using this negative right margin

An alternative solution (which would make more sense for numbers tbh) is to right align the position number; I think I'll do that instead